main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        data = json.load(f)
+
+    for nickname, player_data in data.items():
+        race_data = player_data.get("race")
+        guild_data = player_data.get("guild")
+        skill_data = race_data.get("skills")
+
+        race_data, created = Race.objects.get_or_create(
```

Comment:
Since `created` variable is not used in the code, you could omit it here and on line 30
```suggestion
        race_data, _ = Race.objects.get_or_create(
```